R/utils.R

```diff
@@ -211,11 +212,10 @@ coerce_date <- function(dates) {
 
   if (anyNA(res)) {
     bads <- is.na(res)
-    stop(sprintf(
-      "Failed to parse with `as.IDate`: {%s} (indices {%s}).",
-      toString(dates[bads]),
-      toString(which(bads))
-    ))
+    cli::cli_abort(
```

Comment:
That's a mildly annoying false positive on the linter. I guess mark w/ # no lint after bads?